### PR TITLE
Fix: [v2] Update message receipts on real time basis

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-1620](https://sendbird.atlassian.net/browse/UIKIT-1620)

## Description Of Changes

* Calculate message status in the message content component, not from outside of the component
* Apply onUnreadMemberCountUpdated and onUndeliveredMemberCountUpdated channel handlers
  into the Conversation component

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

### CODE REVIEW

File: src/smart-components/Conversation/hooks/useHandleChannelEvents.js [hunk 1]
```diff
@@ -63,6 +63,26 @@ function useHandleChannelEvents({ currentGroupChannel, sdkInit, hasMoreToBottom
         }
       };
 
+      ChannelHandler.onReadReceiptUpdated = (channel) => {
+        if (compareIds(currentGroupChannel.url, channel.url)) {
```
Comment (sravan-s) [inline src/smart-components/Conversation/hooks/useHandleChannelEvents.js:67]: currentGroupChannel?.url

**Review by sravan-s — APPROVED** [review]

File: src/smart-components/Conversation/hooks/useHandleChannelEvents.js [hunk 2]
```diff
@@ -63,6 +63,26 @@ function useHandleChannelEvents({ currentGroupChannel, sdkInit, hasMoreToBottom
         }
       };
 
+      ChannelHandler.onReadReceiptUpdated = (channel) => {
```
Comment (sravan-s) [inline src/smart-components/Conversation/hooks/useHandleChannelEvents.js:66]: Why do we have to `type: messageActions.SET_CURRENT_CHANNEL` here?

Comment (HoonBaek) [reply src/smart-components/Conversation/hooks/useHandleChannelEvents.js:66]: We need to set the current channel when the receipts change, and SET_CURRENT_CHANNEL exactly do that

File: src/smart-components/Conversation/hooks/useHandleChannelEvents.js (hunk 1 above)
Comment (HoonBaek) [reply src/smart-components/Conversation/hooks/useHandleChannelEvents.js:67]: good

**Pull request merged**